src/context/EarnManagerProvider/providers/yearn/api/yearn-types.ts

```diff
@@ -9,12 +8,11 @@ export type Allowanceinput = {
 }
 
 export type ApproveInput = {
-  bip32Params: BIP32Params
+  accountNumber?: number
```

Comment:
didn't we just ADD `bip32Params`? I don't understand the reason behind this change.

Reply:
in this case we're expecting the default path to be for account 0. If a consumer wants to use another account were able to easily build the new bip32Params for that account number